hack/deploy.sh

```diff
@@ -10,12 +10,8 @@ DEFAULT_OPERAND_IMAGE="quay.io/cincinnati/cincinnati:latest"
 RELATED_OPERATOR_IMAGE="${RELATED_IMAGE_OPERATOR:-${DEFAULT_OPERATOR_IMAGE}}"
 RELATED_OPERAND_IMAGE="${RELATED_IMAGE_OPERAND:-${DEFAULT_OPERAND_IMAGE}}"
 
-if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
-	RELATED_OPERATOR_IMAGE="registry.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:updateservice-operator"
-	GRAPH_DATA_IMAGE="registry.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:updateservice-graph-data-container"
-
-	echo "Openshift CI detected, deploying using image $RELATED_OPERATOR_IMAGE and ${GRAPH_DATA_IMAGE}"
-
+if [ -n "$OPENSHIFT_CI" ]; then
+	echo "Openshift CI detected, deploying using ${RELATED_OPERATOR_IMAGE} as operator and ${RELATED_OPERAND_IMAGE} as operand and ${GRAPH_DATA_IMAGE} as graph data image"
```

Comment:
do we even care about `OPENSHIFT_CI`?  Looks like we could drop this conditional, and just have:

```sh
if ! [ -n "$KUBECONFIG" ]; then
	echo "KUBECONFIG environment variable must be set."
	exit 1
fi
if ! [ -n "$GRAPH_DATA_IMAGE" ]; then
	echo "GRAPH_DATA_IMAGE environment variable must be set."
	exit 1
fi
```

checked in all cases.  Maybe with an informational:

```sh
echo "Deploying using ${RELATED_OPERATOR_IMAGE} as operator and ${RELATED_OPERAND_IMAGE} as operand and ${GRAPH_DATA_IMAGE} as graph data image"
```

for all cases?

Reply:
SGTM, fixed in a802f3b